scripts/lib/mkcloud-libvirt.sh

```diff
@@ -349,6 +349,12 @@ function libvirt_do_setuplonelynodes()
         local lonely_disk
         lonely_disk="$vdisk_dir/${cloud}.node$i"
 
+        lvdisplay $lonely_disk
+        if [ $? -ne 0 ]; then
+            hdd_size=${lonelynode_hdd_size}
+            _lvcreate ${cloud}.node$i $hdd_size $cloudvg
```

Comment:
Well it did not reached `libvirt_do_create_cloud_lvm` for me:
https://gist.github.com/matrixik/0d352db50a9e8760da49911fd88bd3cd